src/components/GlobeOptimized.jsx

```diff
@@ -602,13 +585,28 @@ export default function GlobeOptimized(props) {
     }
   }
 
+  function enableRotation(){
+    setTimeout(() => {
+      timer(function (elapsed) {
+        const { projection } = globeAttributes.current;
+        const rotation = projection.rotate();
+        const degPerMs = globeAttributes.current.rotation_speed;
+        rotation[0] -= 100 * degPerMs * 60;
+        rotation[1] -= 100 * degPerMs * 0;
+        projection.rotate(rotation);
+        globeAttributes.current.rotation = rotation;
+        render();
+      }); }, 2);
+      setTimeout(function(){
+        globeAttributes.current.rotation_speed = 0;
+     }, 5700);
```

Comment:
Curious why do we setup `5700` here? Does `5700` stands for the duration that we're rotating the globe?

Reply:
yes